### DISCUSSION TITLE
本地访问

### DISCUSSION THREAD

**Discussion by likesrt** [messense/aliyundrive-webdav#294] [Q&A]
![截图_20222311032319](https://user-images.githubusercontent.com/31208912/153481451-8c65b8f9-f7ed-4863-99db-6ba0e9efbbfd.png)
本地访问提示连接被拒绝

**Comment by messense** [comment]
请按照 [问题反馈](https://github.com/messense/aliyundrive-webdav/issues/new?assignees=&labels=bug&template=bug_report.yml) 流程提供更多信息。
